pkg/apis/crd/v1alpha2/types.go

```diff
@@ -359,15 +359,24 @@ type IPAddressState struct {
 }
 
 type IPAddressOwner struct {
-	Pod *PodOwner `json:"pod,omitempty"`
+	Pod         *PodOwner         `json:"pod,omitempty"`
+	StatefulSet *StatefulSetOwner `json:"statefulSet,omitempty"`
 }
 
+// Pod owner
 type PodOwner struct {
 	Name        string `json:"name"`
 	Namespace   string `json:"namespace"`
 	ContainerID string `json:"containerID"`
 }
 
+// StatefulSet owner
+type StatefulSetOwner struct {
```

Comment:
Can we move the CRD change to a separate PR and review/merge firstly?

Reply:
ok, done: https://github.com/antrea-io/antrea/pull/3093